Dockerfile

```diff
@@ -1,9 +1,9 @@
 # Build the manager binary
-FROM golang:1.22 as builder
+FROM --platform=${BUILDPLATFORM} golang:1.22 as builder
```

Comment:
"BUILDPLATFORM is part of a set of [automatically defined (global scope) build arguments](https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope) that you can use. It will always match the platform or your current system and the builder will fill in the correct value for us."

from: https://www.docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide/

Putting this here for reference